File: Semestres/11.Febrero-Junio 2022/EstructurasDatosUSB/corte2/Tema21RecorridosArboles/MainTree2.py
from Tree2 import Tree

objTree = Tree(
    1,
    Tree(
        2,
        Tree(4,
            Tree(8,None,None),
            Tree(9,None,None)
        ),
        Tree(
            5,
            Tree(10,None,None),
            Tree(11,None,None)
        )
    ),
    Tree(
        3,
        Tree(
            6,
            Tree(13,None,None),
            None
        ),
        Tree(
            7,
            Tree(14,None,None),
            None)
    )
)

print(objTree.preorder())
print(objTree.inororder())
print(objTree.posorder())

File: Semestres/11.Febrero-Junio 2022/EstructurasDatosUSB/corte2/Tema21RecorridosArboles/MainTree2.java
public class MainTree2 {
    public static void main(String[] args) {
        Tree2 objTree = new Tree2(
                1,
                new Tree2(
                        2,
                        new Tree2(4,
                                new Tree2(8, null, null),
                                new Tree2(9, null, null)),
                        new Tree2(
                                5,
                                new Tree2(10, null, null),
                                new Tree2(11, null, null))),
                new Tree2(
                        3,
                        new Tree2(
                                6,
                                new Tree2(13, null, null),
                                null),
                        new Tree2(
                                7,
                                new Tree2(14, null, null),
                                null)));

        System.out.println(objTree.preorder());
        System.out.println(objTree.inorder());
        System.out.println(objTree.posorder());
    }
}
